syft/src/main/java/org/openmined/syft/execution/JobId.kt

```diff
@@ -0,0 +1,31 @@
+package org.openmined.syft.execution
+
+import java.security.MessageDigest
+
+/**
+ * A uniquer identifier class for the job
+ * @property modelName The name of the model used in the job while querying PyGrid
+ * @property version The model version in PyGrid
+ */
+data class JobId(
+    val modelName: String,
```

Comment:
Maybe make a jobModel class that stores jobId, modelname, model version, plans and protocols.